src/scouter/scanner/ScanningTab.tsx

```diff
@@ -0,0 +1,120 @@
+import { useEffect, useRef, useState } from "react";
+import { useNavigate } from "react-router-dom";
+import "./QRStyles.css";
+import QrScanner from "qr-scanner";
+import React from "react";
+
+const ScanningTab = () => {
+  const navigate = useNavigate();
+  const scanner = useRef<QrScanner>();
+  const videoEl = useRef<HTMLVideoElement>(null);
+  const qrBoxEl = useRef<HTMLDivElement>(null);
+  const [qrOn, setQrOn] = useState<boolean>(false); // Tracks if the QR scanner is on
```

Comment:
comment is not needed
```suggestion
  const [qrOn, setQrOn] = useState<boolean>(false);
```